CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -391,6 +391,19 @@ function PlayerHospital:adviseBoilerBreakdown(broken_heat)
   end
 end
 
+local AnnouncementPriority = _G["AnnouncementPriority"]
```

Comment:
I would recommend always puttting this line at the top of the file before the class declaration. Also missing the ``require``